main/lsp/protocol.cc

```diff
@@ -158,7 +158,9 @@ unique_ptr<Joinable> LSPPreprocessor::runPreprocessor(QueueState &incomingQueue,
 }
 
 optional<unique_ptr<core::GlobalState>> LSPLoop::runLSP() {
-    // Naming convention: thread that executes this function is called coordinator thread
+    multithreadedMode = true;
```

Comment:
do we need it? can we make single/multithreaded mode distinction go away?

Reply:
I only need this distinction to prevent single threaded mode from enabling cancelation.

If single threaded mode enables cancelation, there will be ENFORCE failures because the single threaded path doesn't call `startCommitEpoch`.

Hmm, maybe I can change it so it does? I think I can remove this.

Reply:
Done. I removed multuthreadedMode.